.travis.yml

```diff
@@ -60,7 +79,7 @@ script:
       export MPL_REPO_DIR=$PWD  # needed for pep8-conformance test of the examples
       mkdir ../tmp_test_dir
       cd ../tmp_test_dir
-      gdb -return-child-result -batch -ex r -ex bt --args python ../matplotlib/tests.py -sv --processes=8 --process-timeout=300 $TEST_ARGS
+      gdb -return-child-result -batch -ex r -ex bt --args python ../matplotlib/tests.py -sv --processes=2 --process-timeout=300 $TEST_ARGS
```

Comment:
Should we make this a variable and change the "echo Testing using 8 processes" above so that it will stay in sync?